benchmark/benchmark.py

```diff
@@ -79,14 +97,19 @@ def profile(func, identifier):
 
 
 def cli():
-    "Read the commandline parameters, perform the requested action."
-    # import pdb; pdb.set_trace()
+    """Parse the command line arguments, perform the requested action.
+    """
+    parser = argparse.ArgumentParser(description='PennyLane benchmarking tool')
+    parser.add_argument('--version', action='version', version=__version__)
+    #parser.add_argument('-v', '--verbose', action='store_true', help='verbose mode')
```

Comment:
how come the verbose mode is commented out? :laughing:

Reply:
I'd be in favour of adding verbose mode in a separate PR, just to get this one merged in